src/utils/notifications/requestNotification.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable class-methods-use-this */
+import emitter from '../eventEmitters/emitter';
+import NotificationService from '../../services/notificationService';
+import userService from '../../services/userService';
+import app from '../../index';
+import Emails from '../mails/email';
+import RequestNotification from '../mails/requestNotification.email';
+
+/** Class representing a Notifications . */
+class Notifications {
+  /**
+   * @param {object} request
+   * @return {function} create notifications
+   */
+  async makeNotification(request) {
+    const manager = await userService.findUser({ userRoles: 'Manager' });
+    const { firstName, lastName } = await userService.findUser({ id: request.user });
+    return {
+      id: manager.id,
+      firstName,
+      lastName,
+      emailAllowed: manager.emailAllowed,
+      userEmail: manager.userEmail,
+      manager: manager.firstName
+    };
+  }
+
+  /**
+   * @return {function} create notifications
+   */
+  async requestCreatedNotification() {
+    await emitter.on('request created', async (request) => {
+      const {
+        id,
+        firstName,
+        lastName,
+        userEmail,
+        emailAllowed,
+        manager
+      } = await this.makeNotification(request);
+      const notification = `${firstName} ${lastName} requested a new trip`;
+      const notif = await NotificationService.createNotification({
```

Comment:
Maybe we can name both these notifications appropriately instead of `notif` we can do `inAppNotification`. And then `notification` we can name it `notificationMessage`